packages/vue/src/components/organisms/SfContentPages/_internal/SfContentCategory.vue

```diff
@@ -1,18 +1,18 @@
 <template>
-  <div>
+  <div class="sf-content-category">
     <slot />
   </div>
 </template>
 <script>
 export default {
-  name: "SfContentCategory",
+  name: 'SfContentCategory',
```

Comment:
Please check if you don't have some kind of local linter enabled - it overwrites the project one. We are using double `" "`, so please fix that :)